src/Stripe.net/Services/PreviewService.cs

```diff
@@ -0,0 +1,38 @@
+namespace Stripe.Services
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class PreviewService
+    {
+        private readonly StripeClient client;
+
+        internal PreviewService(StripeClient client)
+        {
+            this.client = client;
+        }
+
+        public Task<StripeResponse> Get(string path, RawRequestOptions requestOptions = null, CancellationToken cancellationToken = default)
```

Comment:
Do we need to call these `GetAsync`, etc. since it calls the async version of `RawRequestAsync`? Should we make sync versions too?